packages/mobile/src/store/offline-downloads/sagas/rehydrateOfflineDataSaga.ts

```diff
@@ -36,6 +37,9 @@ export function* rehydrateOfflineDataSaga() {
   const isOfflineModeEnabled = yield* call(getIsOfflineEnabled)
   if (!isOfflineModeEnabled) return
 
+  // Can remove this after all clients are likely updated
+  yield* migrateOfflineDataPathSaga()
```

Comment:
do we want some analytics around this to make a best guess? no worries but a thought.

Reply:
yeah I was thinking the same thing. Maybe one for success and one for error? When we stop seeing both for a while we're likely good to rip it out.